hardhat.config.ts

```diff
@@ -92,6 +92,11 @@ export default {
     hardhat: {
       blockGasLimit: 12.5e6,
     },
+    localhost: {
+      blockGasLimit: 12.5e6,
+      url: "http://127.0.0.1:8545",
+    },
```

Comment:
AFAIU this shouldn't be needed as `localhost` network should be configured by default.